client/reader/post-byline/index.jsx

```diff
@@ -72,7 +73,8 @@ var PostByline = React.createClass( {
 					<a className="reader-post-byline__date-link"
 						onClick={ this.recordDateClick }
 						href={ post.URL }
-						target="_blank"><PostTime date={ post.date } /></a>
+						target="_blank"
+						icon={ true }><PostTime date={ post.date } />{ this.props.icon ? <Gridicon icon="external" size={ 14 } /> : null }</a>
```

Comment:
Nope, this is a faux external link, because it does different things in stream and out of stream. Should just drop the icon prop.